lms/djangoapps/courseware/grades.py

```diff
@@ -459,7 +459,26 @@ def progress_summary(student, request, course, field_data_cache=None, scores_cli
     in case there are unanticipated errors.
     """
     with manual_transaction():
-        return _progress_summary(student, request, course, field_data_cache, scores_client)
+        progress = _progress_summary(student, request, course, field_data_cache, scores_client)
+        if progress:
+            return progress['chapters']
+        else:
+            return None
```

Comment:
_progress_summary can return None if the module is not found (line 499 in the original file). For safety, I figured it best to handle that case in case it came up.